.github/workflows/releaser.yaml

```diff
@@ -0,0 +1,47 @@
+name: Releaser
+
+on:
+  workflow_dispatch:
+    inputs:
+      base_branch:
+        description: 'The base branch to release from'
+        required: true 
+
+permissions:
+  contents: write
+
+jobs:
+  releaser:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set branch
+        run: echo "BRANCH=${{ github.event.inputs.base_branch }}" >> $GITHUB_ENV
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ env.BRANCH }}
+      - name: Fetch all tags
+        run: git fetch --force --tags
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+      - name: Release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GHACTION_GH_PAT }}
+          DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+          DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+          QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
+          QUAY_TOKEN: ${{ secrets.QUAY_TOKEN }}
+          HOMEBREW_CORE_REPO: ${{ secrets.HOMEBREW_CORE_REPO }}
+          OCI_ORG: ${{ secrets.OCI_ORG }}
+          OPERATOR_HUB_REPO: "k8s-operatorhub/community-operators"
+          TAG: "1"
+        run: |
+          git config --global user.email "github-action@noobaa.io"
+          git config --global user.name "NooBaa GitHub Action"
+
+          bash build/tools/builder.sh --oci-org $OCI_ORG --gh-org noobaa --gh-repo noobaa-operator || exit 1
+          bash build/tools/releaser.sh --oci-org $OCI_ORG --gh-org noobaa --gh-repo noobaa-operator || exit 1
+      # - name: "Krew Release"
+      #   uses: rajatjindal/krew-release-bot@v0.0.40
```

Comment:
Why do we need those comments? 
If we will not use it, we can remove it

Reply:
We will need it. I have commented it out to because we will need to create first PR in the krew repository manually and only after that this bot can be used (limitation of the krew bot). I left it here as a comment to highlight that this needs to be added in the workflow so that others are aware of it.